tests/foreman/ui_airgun/test_reporttemplates.py

```diff
@@ -0,0 +1,280 @@
+"""Test for Report Templates
+
+:Requirement: Reports
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+
+from nailgun import entities
+
+from robottelo.decorators import (
+    fixture,
+    tier2,
+    stubbed,
+)
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@tier2
+@stubbed
+def test_positive_create_report(session):
+    """ Create a report template
+
+    :id: 906ac1f8-c02c-4197-9c98-01e8b9f2f075
+
+    :setup: User with reporting access rights
+
+    :steps:
+
+        1. Monitor -> Report Templates -> Create Template
+        2. Insert any content
+        3. Submit
+
+    :expectedresults: The report template should be created
+
+    :CaseImportance: High
+    """
+
+
+@tier2
+@stubbed
+def test_positive_read_report(session):
+    """ Create a report template
+
+    :id: 906ac1f8-c02c-4197-9c97-01e8b9f2f075
+
+    :setup: User with reporting access rights, some existing report template
+
+    :steps:
+
+        1. Monitor -> Report Templates
+        2. Select template
+
+    :expectedresults: The report template's content is shown
+
+    :CaseImportance: High
+    """
+
+
+@tier2
+@stubbed
+def test_positive_update_report(session):
+    """ Edit a report template
+
+    :id: 906ac1f8-c02c-4297-9c98-01e8b9f2f075
+
+    :setup: User with reporting access rights, some existing report template that is not locked
+
+    :steps:
+
+        1. Monitor -> Report Templates
+        2. Select template
+        2. Change template content
+        3. Submit
+
+    :expectedresults: The report template should be updated
+
+    :CaseImportance: High
+    """
+
```

Comment:
I don't see a test for previewing. It would be good to cover the fact that load macros, e.g. load_hosts in preview mode only lists 10 records to not overwhelm the server.